### DISCUSSION TITLE
How do you set the document.title for web rendered pages?

### DISCUSSION THREAD

**Discussion by nikkwong** [expo/router#267] [Q&A] [answered]
Sorry if this is a silly place to ask—but I couldn't find this documentation anywhere else. Where can we set the page title? I'm using expo-router and have web rendering as a target. Thanks.

**Comment by bencreynolds** [comment, marked as answer]
<Stack.Screen options={{ title: "Example Title" }}/>

**Reply by EvanBacon** [reply]
Correct, we may add a more web-specific API in the future for SEO purposes.

**Reply by Vimiso** [reply]
Doesn't this simply update the title of the header, eg. if you're in a stack or whatever? It doesn't seem to update the actual document title, and sometimes they may need to differ. In the [docs](https://expo.github.io/router/docs/migration/react-navigation/navigation-container/#documenttitle) it mentions `documentTitle` is not supported :(

Really, I just want to wrap the entire web app with one `documentTitle`, syntax would be:
```jsx
<Stack screenOptions={{ headerShown: false, documentTitle: 'App' }} />
```

PS:

The reason I'm not using `expo-router/head` is because in native it throws an error and insists:

```Error: Expo Head: Add the handoff origin to the Expo Config (requires rebuild). Add the Config Plugin { plugins: [["expo-router", { origin: "...<URL>..." }]] }```

...which is for handover purposes I believe, so if I do not wish to configure that, why should it care?

**Reply by vendramini** [reply]
@Vimiso did you find a solution?

**Reply by Vimiso** [reply]
Nope, unfortunately now. In the meantime, opted for adding this to my root layout:

```jsx
{ Platform.OS === 'web' && (
    <Head>
      <title>App name</title>
    </Head>
) }
```
